blueprints/showbie-tooling/files/.eslintrc.js

```diff
@@ -13,10 +16,17 @@ module.exports = {
     semi: ['error', 'always'],
 
     // Custom Ember rules.
-    'ember/new-module-imports': 'error',
-    'ember/no-attrs-in-components': 'error',
-    'ember/no-duplicate-dependent-keys': 'error',
-    'ember/no-global-jquery': 'error',
-    'ember/require-super-in-init': 'error',
+    'ember/no-empty-attrs': 'warn',
+    'ember/no-observers': 'warn',
+    'ember/use-ember-get-and-set': 'warn',
+
+    'ember/order-in-components': 'warn',
+    'ember/order-in-controllers': 'warn',
+    'ember/order-in-models': 'warn',
+    'ember/order-in-routes': 'warn',
+  },
+  globals: {
```

Comment:
I think these should be omitted from the blueprint—seems like it should be application specific.

Reply:
Makes sense to me — is there a way to have the blueprint merge with existing files, or does that just need to be done manually?

Reply:
It'll have to be done manually via the standard Ember CLI diffing tool.